Tests/MSFT_xWindowsOptionalFeature.Tests.ps1

```diff
@@ -0,0 +1,311 @@
+<# 
+.summary
+    Test suite for MSFT_xWindowsOptionalFeature.psm1
+#>
+[CmdletBinding()]
+param()
+
+Import-Module $PSScriptRoot\..\DSCResources\MSFT_xWindowsOptionalFeature\MSFT_xWindowsOptionalFeature.psm1
```

Comment:
@iainbrighton Are you going to merge the test template?  Or do you want me to merge as is?